src/meinberlin/meinberlin/static/stylesheets/scss/components/_badges_theme.scss

```diff
@@ -0,0 +1,31 @@
+/*doc
+---
+title: badges
+name: badges
+category: Type
+---
+
+Badges for labeling if a proposal has been realized or not.
+
+```html_example
+<span class="has-badge is-realized">Is Realized</span>
+<span class="has-badge is-not-realized">Is Not Realized</span>
+```
+
+*/
+.has-badge {
+    @include inline-block;
+    color: $color-text-inverted;
+    font-size: $font-size-small;
+    font-variant: small-caps;
+    padding: 0 5px;
+    text-transform: uppercase;
```

Comment:
The combination of `small-caps` and `uppercase` does not make much sense. I propose to skip `small-caps`.